src/OSPSuite.Core/Domain/Builder/SimulationConfiguration.cs

```diff
@@ -35,38 +44,47 @@ public class SimulationConfiguration : IVisitable<IVisitor>
       public virtual void AddCalculationMethod(ICoreCalculationMethod calculationMethodToAdd) => _allCalculationMethods.Add(calculationMethodToAdd);
 
       public virtual IReadOnlyList<ISpatialStructure> SpatialStructures => all(x => x.SpatialStructure);
```

Comment:
The only unknown is this one here. Not sure how to merge multiple Spatial Structure....